global.json

```diff
@@ -0,0 +1,5 @@
+{
+  "sdk": {
+    "version": "2.0.0"
+  }
+}
```

Comment:
Please add a newline at the end of the file.

Reply:
Done.  BTW the original file was simply the output of running:
```
dotnet new global.json --sdk-version 2.0.0
```